### ISSUE TITLE
It says I don't own Minecraft when I try to log into my Microsoft account

### ISSUE THREAD

**Issue by LoocyGoosey** [killpowa/Twickt-Launcher#1318]
**Describe the bug**
When i go to log into the gdlauncher using my microsoft account it says i need to buy minecraft despite having bought and own the game for years.

**To Reproduce**
1.Attempt to login via Microsoft account

**Expected behavior**

**Possible solutions**
Some ideas of how you would solve this issue, if possible

**Screenshots**

**Operating System:**
- OS: Windows
- Version: 10
- Java Version: 8
- Minecraft Version: 1.16.5
- Mod Loader: Forge

**Additional context**

**Comment by Eskaan** [comment]
Which way did you buy the game, is you account migrated and did you log into https://minecraft.net at least once?

**Comment by wizzah2** [comment]
I'm having the same issue. It says I didn't buy the game, but I did. I purchased it through the Microsoft Store under the "Minecraft for Windows Starter Pack" and I have also logged in to https://minecraft.net/ and I still get the error in the launcher.

**Comment by Eskaan** [comment]
Another question, can you launch the Java edition through the native launcher?

**Comment by wizzah2** [comment]
Yes, I can launch the Java edition through the official launcher that came with the purchase.

**Comment by Alvareto136** [comment]
I have the same problem and it solves itself 2-3 days later when it suddenly let's me log back in.

**Comment by FredericLueck** [comment]
I have this error all the time
First I get a Invalid Session ID Error and cant join servers
My solution was to remove my account and add it again
That worked once
After that I would always have this error and would need to restart my whole PC for it to work again
